explore/src/main/scala/explore/targeteditor/AladinCell.scala

```diff
@@ -94,18 +95,23 @@ case class AladinCell(
 }
 
 trait AladinCommon:
-  given Reusability[Asterism] = Reusability.by(x => (x.toSiderealTracking, x.focus.id))
-  given Reusability[AgsState] = Reusability.byEq
+  val asReu: Reusability[Asterism] = Reusability.by(x => (x.toSiderealTracking, x.focus.id))
+  given Reusability[Asterism]      = asReu.logNonReusable
```

Comment:
Do we want to keep the log in production?

Reply:
Nope, I'll remove it